tests/modules/images/fetchpriority/fetchpriority-tests.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * Tests the adding of fetchpriority to img tags in the content.
+ *
+ * @Covers ::wp_filter_content_tags
+ */
+
+class Fetchpriority_Test extends WP_UnitTestCase {
+	protected static $post;
+	protected static $attachment_id;
+
+	protected $current_size_filter_data   = null;
+	protected $current_size_filter_result = null;
+
+	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+		self::$post = $factory->post->create_and_get();
+
+		$file                = TESTS_PLUGIN_DIR . '/tests/testdata/modules/images/leafs.jpg';
+		self::$attachment_id = $factory->attachment->create_upload_object(
+			$file,
+			self::$post->ID,
+			array(
+				'post_mime_type' => 'image/jpeg',
+			)
+		);
+	}
```

Comment:
Should remove this as we get `$attachment_id` from other code. Check other comment.

Reply:
code removed